services-camel/src/test/resources/config.properties

```diff
@@ -43,8 +43,12 @@ cdr.stream.camel=direct-vm:index.start
 cdr.solrupdate.stream=direct-vm:index.start
 cdr.solrupdate.stream.camel=direct-vm:index.start
 
+cdr.triplesupdate.stream=direct-vm:index.start
```

Comment:
what's the difference between these two properties? do we really need both?

Reply:
the `.stream` versus `.stream.camel` properties? In this environment they are the same, but in most environments they are different.